backend/resolver/resolver.go

```diff
@@ -38,7 +42,7 @@ type Resolver interface {
 	Resolve(ctx context.Context, typeURL string, input proto.Message, limit uint32) (*Results, error)
 	// ValidateResolveInput(typeURL string, input proto.Message) for async validation from frontend
 
-	// Autocomplete(query string) (map[string][]proto.Message, error)
+	AutoComplete(ctx context.Context, typeURL, search string) ([]string, error)
```

Comment:
[]string is too limited. we may want to show additional metadata about a selection in the autocomplete pane.

![Screenshot from 2021-02-08 12-22-05](https://user-images.githubusercontent.com/4712430/107264191-4da6ed00-6a08-11eb-8c76-410b402f4b4b.png)

Reply:
Yeah true good call, let me the ability to add more information.